src/services/LeafValueCoder.ts

```diff
@@ -1,8 +1,11 @@
 import BigNumber from 'bignumber.js';
-import { FIXED_NUMBER_PREFIX, NUMERIC_MULTIPLIER } from '../constants';
+import { FIXED_NUMBER_PREFIX, NUMERIC_MULTIPLIER, INT_PREFIX, MAX_UINT224_HEX } from '../constants';
 import { utils } from 'ethers';
 import { evenHex, prepend0x } from '../utils/helpers';
 import { FeedValue } from '../types/Feed';
+const maxUint224 = BigInt(MAX_UINT224_HEX);
+const maxInt224 = (maxUint224 - 1n) / 2n;
+const minInt224 = (maxInt224 + 1n) * -1n;
```

Comment:
Maybe you can move those into "../constants". Looks like you use these same values in other files.

Reply:
done